frontend/src/app/applications/temporary-outfitters/temp-outfitter-left-nav.component.html

```diff
@@ -0,0 +1,53 @@
+&nbsp;
+  <ul id="temp-outfitter-nav" class="usa-sidenav-list" [style.bottom]="bottom" [style.top]="top" [style.position]="position">
+    <li [ngClass]="getGroupStatus(applicationForm.controls.applicantInfo, applicantInfoErrors)">
```

Comment:
It seems like these `li` could be made into a more abstract component  so that we can leverage creating new navs on other forms and update this one if need be.